lib/node_runner.go

```diff
@@ -28,6 +29,44 @@ var (
 	MaxTransactionsInBallot     int           = 1000
 )
 
+var DefaultHandleMessageFromClientCheckerFuncs = []sebakcommon.CheckerFunc{
+	CheckNodeRunnerHandleMessageTransactionUnmarshal,
+	CheckNodeRunnerHandleMessageHasTransactionAlready,
+	CheckNodeRunnerHandleMessageHistory,
+	CheckNodeRunnerHandleMessagePushIntoTransactionPool,
+	CheckNodeRunnerHandleMessageTransactionBroadcast,
+}
+
+var DefaultHandleBaseBallotCheckerFuncs = []sebakcommon.CheckerFunc{
+	CheckNodeRunnerHandleBallotUnmarshal,
+	CheckNodeRunnerHandleBallotNotFromKnownValidators,
+	CheckNodeRunnerHandleBallotAlreadyFinished,
+}
+
+var DefaultHandleINITBallotCheckerFuncs = []sebakcommon.CheckerFunc{
```

Comment:
what about merge together 
```DefaultHandleInitBallotCheckerFuncs, DefaultHandleSignBallotCheckerFuncs, DefaultHandleAcceptBallotCheckerFuncs, ``` ??

Reply:
end of consensus checker funcs there are some difference ```checknoderunneracceptballot``` like. but I think it's better put those end of method action. isn't it?

Reply:
@cmcm2222
> what about merge together
DefaultHandleInitBallotCheckerFuncs, DefaultHandleSignBallotCheckerFuncs, DefaultHandleAcceptBallotCheckerFuncs, ?? 
end of consensus checker funcs there are some difference checknoderunneracceptballot like. but I think it's better put those end of method action. isn't it?

These are not fit this pr. It is appropriate in pr https://github.com/bosnet/sebak/pull/240/files
It would be really helpful to review only [comparing link](https://github.com/spikeekips/sebak/compare/new-proposer-bug-fixes...Charleslee522:noderunner-message-checker) in here.

Reply:
@Charleslee522 Alright.

Reply:
Can I ask why it's not suitable in this PR?
I have no idea.

Reply:
@cmcm2222 Good question.
Because these codes are not changed in this pr. It is changed in #240.

The reason that the code is included in the `File Changed` is that the branch to be merged with this PR is `bosnet:new-proposer`, but actually this pr is based on `spikeekips:new-proposer-bug-fixes`.

Therefore, you should check diff in https://github.com/spikeekips/sebak/compare/new-proposer-bug-fixes...Charleslee522:noderunner-message-checker not `File Changed`.